build.sbt

```diff
@@ -102,6 +102,8 @@ lazy val commonSettings = Def.settings(
   ),
   Compile / javacOptions ++= javacSettings,
   Test / javacOptions ++= javacSettings,
+  // Akka brings in 0.9.0, but we want 1.0.0:
+  libraryDependencySchemes += "org.scala-lang.modules" %% "scala-java8-compat" % "always",
```

Comment:
Next akka minor release upgrades to scala-java8-compat 1.0.0 already. I opened an issuse to eventually remove that line again: https://github.com/playframework/play-ws/issues/601